### ISSUE TITLE
Authorization is lose in headless

### ISSUE THREAD

**Issue by bazuker** [go-rod/rod#894]
Rod Version: v0.113.3

## The code to demonstrate your question

I have authenticate myself on GitHub. When I use the code below in headless mode I am not authenticated as it seems like the stored session (cookie) is not passed.
When I run the code with `Headless(false)`, it works fine and I am authenticated.

```go
	l := launcher.New().
		UserDataDir("/Users/bazuker/go/src/project/userdata").
		Headless(true).
		Devtools(false)

	url := l.MustLaunch()

	browser := rod.New().
		ControlURL(url).
		Trace(true).
		SlowMotion(1 * time.Second).
		MustConnect()

browser.MustPage("https://github.com").MustWaitStable()
```

## What you got

I am not authenticated in headless mode

## What you expected to see

To be authenticated as I am returning to this website in headless mode.

**Comment by ysmood** [comment]
Yes, it's a well known issue of chromium, you can ask chromium team to fix it.
